README.md

```diff
@@ -161,37 +158,17 @@ shared dependencies in package.json
 
 if this command succeeds, you should have fair confidence the code is in a good working state.
 
-### Adding a new package
-
-To add a new package edit the `lerna.json` file with your new package run
-
-```bash
-npm run sync:dependencies
-```
```

Comment:
Was this for managing the monorepo? I don't remember using this before.

Reply:
Not sure, in any case that command isn't in package.json anymore, so it might have been something we've already deprecated.